mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping

```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/unmapped/C_lygsomtd net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly
+	METHOD m_aecmvuxy of (Lnet/minecraft/unmapped/C_ogavsvbr;FLnet/minecraft/unmapped/C_hvilmtvi;)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 entityType
+		ARG 1 chance
+		ARG 2 range
+	METHOD m_ipziivps (Lnet/minecraft/unmapped/C_usxaxydn;FLnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 2 targetLivingEntity
+	METHOD m_nozcjkxz (Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;Lnet/minecraft/unmapped/C_ujlmiamh;Ljava/util/function/Predicate;FLnet/minecraft/unmapped/C_lygsomtd$C_gvxfhlod;Lnet/minecraft/unmapped/C_ujlmiamh;Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_usxaxydn;J)Z
+		ARG 7 livingEntiity
+	METHOD m_nteqkjjv (Lnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 0 predicate
+	METHOD m_ouvkkubu of (FLnet/minecraft/unmapped/C_hvilmtvi;)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 chance
+		ARG 1 range
+	METHOD m_shrczqhx (Lnet/minecraft/unmapped/C_ogavsvbr;Lnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 1 entity
+	METHOD m_swcvkveo (Ljava/util/function/Predicate;FLnet/minecraft/unmapped/C_lygsomtd$C_gvxfhlod;Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;)Lcom/mojang/datafixers/kinds/App;
+		ARG 3 behaviour
```

Comment:
```suggestion
		ARG 3 behavior
```
britishism, I need to poke the dictionary about this one it shouldn't been allowed